.gitignore

```diff
@@ -10,6 +10,7 @@ yarn-error.log*
 # testing
 coverage/
 stats.html
+.run/
```

Comment:
Notes: This folder is auto-generated by PHP Storm

Reply:
NITS: Can I ask you to move it to the `# editor` section after line 29?